### PR TITLE
Updates to redirect file

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updates to conductor.yml file to address some missing and broken redirects
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* n/a

### CODE REVIEW

File: packages/@okta/vuepress-site/conductor.yml [hunk 1]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/ios/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/ios/main/
+  - from: /docs/guides/sign-into-mobile-app/ios/create-okta-application
+    to: /docs/guides/build-custom-ui-mobile/ios/main/
+  - from: /docs/guides/sign-into-mobile-app/android/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/android/create-okta-application
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/react-native/create-okta-application/index.html
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/conductor.yml:1380]: exact dup of line 2896 (goes to same location)

File: packages/@okta/vuepress-site/conductor.yml [hunk 2]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/#create-an-okta-application
+  - from: /docs/guides/sign-into-mobile-app/react-native/create-okta-application
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/conductor.yml:1382]: dup of line 2876 (goes to same location)

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
Found a few duplicates:
- the exact duplicates that go to the same location can be removed
- the duplicates that go to different locations need to be assessed to remove the non-correct redirect

File: packages/@okta/vuepress-site/conductor.yml [hunk 3]
```diff
@@ -1362,9 +1362,41 @@ redirects:
   - from: /guides/sign-in-with-microsoft/index.html
     to: /docs/guides/sign-in-with-microsoft/
   - from: /guides/sign-into-mobile-app/index.html
-    to: /docs/guides/sign-into-mobile-app-redirect/
+    to: /docs/guides/build-custom-ui-mobile/android/main/
   - from: /guides/sign-into-mobile-app
-    to: /docs/guides/sign-into-mobile-app-redirect/
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/android/main/
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1369]: Agreed. I fixed this locally.

File: packages/@okta/vuepress-site/conductor.yml [hunk 4]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+  - from: /docs/guides/sign-into-mobile-app/create-okta-application
+    to: /docs/guides/build-custom-ui-mobile/android/main/
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1371]: Agreed. Fixed locally.

File: packages/@okta/vuepress-site/conductor.yml [hunk 5]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+  - from: /docs/guides/sign-into-mobile-app/ios/create-okta-application/index.html
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1372]: Good catch. Line 1372 is the correct To: url goes to the redirect guide, which I believe is incorrect. Let me confirm with Chris.

[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1372]: After further review, the "sign-into-mobile-app" redirects to need to go to the "redirect" docs. And this is duplicated. I deleted entry 1372 and 1373

File: packages/@okta/vuepress-site/conductor.yml [hunk 6]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/ios/main/
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1373]: Yes, the to: line is correct in 2836. Removing duplicate (line 1372-73)

File: packages/@okta/vuepress-site/conductor.yml [hunk 7]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+  - from: /docs/guides/sign-into-mobile-app/ios/create-okta-application
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1374]: deleted duplicate. Thanks.

File: packages/@okta/vuepress-site/conductor.yml [hunk 8]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/ios/main/
+  - from: /docs/guides/sign-into-mobile-app/android/create-okta-application/index.html
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1376]: Thanks. Deleted duplicates at these line numbers.

File: packages/@okta/vuepress-site/conductor.yml [hunk 9]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/android/create-okta-application
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1378]: Thanks. `sign-into-mobile-app-redirect` is the correct guide. Deleted duplicate.

File: packages/@okta/vuepress-site/conductor.yml (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1380]: Deleted duplicates here.

File: packages/@okta/vuepress-site/conductor.yml (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1382]: I've deleted these redirects as they're supposed to go to a guide that does not exist yet.

File: packages/@okta/vuepress-site/conductor.yml [hunk 10]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/react-native/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/#create-an-okta-application
+  - from: /docs/guides/sign-into-mobile-app/react-native/create-okta-application
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/android/create-an-okta-application/index.html
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1384]: Deleted this. There's no redirect React Native guide yet.

File: packages/@okta/vuepress-site/conductor.yml [hunk 11]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/android/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/android/create-an-okta-application
+    to: /docs/guides/build-custom-ui-mobile/android/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/ios/create-an-okta-application/index.html
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1388]: Deleted

File: packages/@okta/vuepress-site/conductor.yml [hunk 12]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/ios/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/ios/create-an-okta-application
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1390]: Deleted. Thanks.

File: packages/@okta/vuepress-site/conductor.yml [hunk 13]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/ios/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/ios/create-an-okta-application/index.html
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1392]: Deleted

File: packages/@okta/vuepress-site/conductor.yml [hunk 14]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/ios/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/ios/create-an-okta-application
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1394]: Deleted

File: packages/@okta/vuepress-site/conductor.yml [hunk 15]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/ios/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/react-native/create-an-okta-application/index.html
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1396]: Deleted

File: packages/@okta/vuepress-site/conductor.yml [hunk 16]
```diff
@@ -1362,9 +1362,41 @@ redirects:
...
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/#create-an-okta-application
+  - from: /docs/guides/build-custom-ui-mobile/react-native/create-an-okta-application
```
[the remark this answers is not in the document]
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/conductor.yml:1398]: Deleted

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚢

**Comment by brianduffield-okta** [comment]
did a quick double-check on staging and all looks good. Merging.

**Pull request merged**